web/models/email.ex

```diff
@@ -41,6 +43,7 @@ defmodule HexWeb.Email do
 
   def verify(email) do
     change(email, %{verified: true, verification_key: nil})
+    |> unique_constraint(:email, name: "emails_email_key", message: "Email already verified by another user")
```

Comment:
I don't think we need to mention "verified" here or above. It's enough to say "email already in use".